solutions/security/explore/configure-dns-histogram.md

```diff
@@ -0,0 +1,47 @@
+---
+applies_to:
+  stack: ga
+  serverless:
+    security: ga
+products:
+  - id: security
+  - id: cloud-serverless
+---
+
+# Configure the DNS histogram
+
+The DNS histogram (**Top domains by dns.question.registered_domain**) on the **Network** page helps you visualize domain activity in your environment. Depending on your {{kib}} setup—for example, if you're using {{elastic-defend}}—you may need to add the `dns.question.registered_domain` field so that DNS data appears correctly.
+
+If the DNS histogram is empty, follow these steps to populate the data.
+
+## Add the `dns.question.name` field
+
+Add the `dns.question.name` field to the Events table to confirm that DNS data is available.
+
+1. On the **Network** page, select the **Events** tab.
```

Comment:
Would the user typically know that the Network page is under **Explore** in the main nav? Would it makes sense to give a full **Explore > Network** path? Or maybe use the language about the global search field?